src/components/TokenCard/TokenCard.tsx

```diff
@@ -1,29 +1,31 @@
 import React from 'react';
 import { View } from 'react-native';
 import { TouchableOpacity } from 'react-native-gesture-handler';
-import { TokenType } from '@styles';
-import { useTheme, useLanguage } from '@hooks';
+import { useTheme, useTokens } from '@hooks';
+import { numberFormat } from '@helpers/utilities';
 import { TokenCardProps } from './TokenCard.types';
 import { useTokenCard } from './TokenCard.hooks';
 import { makeStyles } from './TokenCard.styles';
 import Text from '../Text/Text';
 import Icon from '../Icon/Icon';
-import Token from '../Token/Token';
-import Input from '../Input/Input';
+import TokenInputInner from '../TokenInputInner/TokenInputInner';
+import { InterestTag } from './InterestTag/InterestTag';
+import { SelectorHeader } from './SelectorHeader/SelectorHeader';
 
 const TokenCard: React.FC<TokenCardProps> = ({
 	token,
 	onPress,
 	balance,
-	innerRef,
 	disableMax = false,
 	updateQuotes,
 	conversionAmount = '',
-	notTouchable = false
+	notTouchable = false,
+	apy,
+	tokenBalance
 }) => {
 	const { colors } = useTheme();
 	const styles = makeStyles(colors);
-	const { i18n } = useLanguage();
+	const { tokens } = useTokens();
```

Comment:
I don't think you should use it here. The USD balance can be passed into this component.